projects/plugins/jetpack/_inc/lib/plugins.php

```diff
@@ -70,7 +69,11 @@ public static function install_plugin( $slug ) {
 
 		$result = $upgrader->install( $zip_url );
 
+		// Will use ->stat() for recording success/fails.
+		$jetpack = Jetpack::init();
```

Comment:
Great suggestions, thanks. Initially didn't add stats for activation because I suspect that will fail much less often. But it doesn't hurt to add nonetheless. d5f9bde

Reply:
Wait, I went to far with adding things in install_and_activate. brb